src/main/resources/i18n/messages_uk.properties

```diff
@@ -0,0 +1,15 @@
+# General messages
```

Comment:
I'm not against UA language :)
But why do we need this for API ?

Reply:
Localization is a "nice to have" feature for modern global apps.
- [RESTful API and i18n: how to design the response?](https://softwareengineering.stackexchange.com/questions/340469/restful-api-and-i18n-how-to-design-the-response)
- [Best practice regarding REST services and I18N](https://stackoverflow.com/questions/6544077/best-practice-regarding-rest-services-and-i18n)

The same question for accessibility: do we need it? Yes, always! 🤗